pkgs/applications/audio/clerk/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, makeWrapper, rofi, mpc_cli, perl, utillinux, pythonPackages }:
+
+stdenv.mkDerivation {
+  name = "clerk";
```

Comment:
Is there a reason not to use a released version? https://github.com/carnager/clerk/
Please use also include the version number in the name or suffix with "-unstable-13-10-2016" for unstable versions.